src/index.js

```diff
@@ -1,9 +1,15 @@
 import { parse } from "babylon-lightscript";
 import { defaultImports, lightscriptImports, lodashImports } from "./stdlib";
 
+import { arrayStatementVisitor, objectStatementVisitor } from "./for-in";
+
 export default function (babel) {
   const { types: t } = babel;
 
+  // XXX: don't love this, but scrolling through a 2000 line file is maddening.
+  const forInArrayStatementVisitor = arrayStatementVisitor(babel);
```

Comment:
Hah, yeah... Thanks for doing so, I'll start thinking of a better organization strategy. 

The fact that `babel` essentially has to come in through dependency injection makes it a real pain... regardless, will do in a follow-up. Probably also will convert to use lightscript soon too.